client/src/components/landing/LoginButton.vue

```diff
@@ -3,14 +3,14 @@
     <router-link
       v-if="isLoggedIn"
       :to="{ name: 'MistakesPage' }"
-      class="btn btn-primary"
+      class="btn btn-text mj-login-button"
```

Comment:
Something like `btn-text` class does not exist. I don't like the idea to override buttons with such way, that you add just border and color. Button is a complex element in HTML and it has its own states (hover, focus, active, active:focus, etc.). By just adding border and color we lose the user experience of accesing the button that bootstrap provides. I would use class `btn-outline-primary`.

In order for it to work properly, add to mj-button.scss those lines:

```

$button-outline-variants: (
  'primary': (
    'regular': '700',
    'focus': '900',
    'hover': '900',
    'active': '500',
  ),
  'secondary': (
    'regular': '700',
    'focus': '900',
    'hover': '900',
    'active': '500',
  ),
  'error': (
    'regular': '700',
    'focus': '900',
    'hover': '900',
    'active': '500',
  ),
);
```
at line 28, and after you add this, at line 76:
```
@each $variant, $states in $button-outline-variants {
    &.btn-outline-#{$variant} {
      color: mistakes-journal.color($variant, map.get($states, 'regular'));

      &:hover {
        background-color: mistakes-journal.color($variant, map.get($states, 'hover'));
        color: mistakes-journal.color('gray', '0');
      }

      &:focus {
        background-color: mistakes-journal.color($variant, map.get($states, 'focus'));
        color: mistakes-journal.color('gray', '0');

      }

      &.active,
      &:active {
        background-color: mistakes-journal.color($variant, map.get($states, 'active'));
        color: mistakes-journal.color('gray', '0');
      }
    }
  }
```